Image.js

```diff
@@ -1,46 +1,46 @@
 module.exports = {
-  list: {
+  providers: {
     // DummyImage
     DummyImage: {
-      getImgUrl: function (size) {
+      getImageUrl: function (size) {
         'use strict';
         return 'https://dummyimage.com/'+size+'/000/fff';
       }
     },
     // LoremPicsum
     LoremPicsum: {
-      getImgUrl: function (size) {
+      getImageUrl: function (size) {
         'use strict';
         var [width,height]=size.split('x');
         return 'https://picsum.photos/'+width+'/'+height+'/?random';
       }
     },
     // FakeImg
     FakeImg: {
-      getImgUrl: function (size) {
+      getImageUrl: function (size) {
         'use strict';
         var [width,height]=size.split('x');
         return 'https://fakeimg.pl/'+width+'x'+height+'/384f66/ecf0f1/?text=Spaceholder&font=lobster';
       }
     }
   },
-  prvd: 'random',
-  getPrvd: function () {
+  currentProvider: 'random',
+  getProvider: function () {
     'use strict';
     // Select random image provider
-    if (this.prvd==='random') {
-      var l=Object.keys(this.list);
+    if (this.provider==='random') {
```

Comment:
You forgot to rename it here and below. This works somehow, so there might be an issue with the tests.
```suggestion
    if (this.currentProvider==='random') {
```